Makefile

```diff
@@ -132,6 +132,11 @@ build-itest:
 	$(GOBUILD) -tags="$(ITEST_TAGS)" -o lnd-itest $(ITEST_LDFLAGS) $(PKG)/cmd/lnd
 	$(GOBUILD) -tags="$(ITEST_TAGS)" -o lncli-itest $(ITEST_LDFLAGS) $(PKG)/cmd/lncli
 
+build-itest-windows:
+	@$(call print, "Building itest lnd and lncli.")
+	$(GOBUILD) -tags="$(ITEST_TAGS)" -o lnd-itest.exe $(ITEST_LDFLAGS) $(PKG)/cmd/lnd
```

Comment:
can't windows star binaries without the `.exe` file ending?

Reply:
No, unfortunately not. The missing `.exe` suffix was part of the problem why the executable wasn't found during execution. There's an environment variable called `PATHEXT` which lists all allowed extensions (like `.exe`, `.cmd`, `.com` and so on). This allows you to type a command in the command line without specifying the extension. Then it just tries to find a file with one of the suffixes in the env variable. But the target binary still has to have one of those extensions. Can't make it work unix style unfortunately.